games-puzzle/neverball/neverball-1.6.0~20190609.recipe

```diff
@@ -1,13 +1,17 @@
-SUMMARY="Clone of Super Monkey Ball using SDL/OpenGL"
-DESCRIPTION="Tilt the  floor to roll a  ball through an obstacle  course within the \
-given  time.  If  the  ball falls  or time  expires, a ball is lost. \
-Collect coins to unlock the exit and earn extra balls."
+SUMMARY="Clone of Super Monkey Ball using SDL2/OpenGL"
+DESCRIPTION="Tilt the floor to roll a ball through an obstacle course within \
+the given time. If the ball falls or time expires, a ball is lost. \
+Collect coins to unlock the exit and earn extra balls. Red coins are worth 5. \
+Blue coins are worth 10. A ball is awarded for 100 coins."
 HOMEPAGE="https://neverball.org"
-COPYRIGHT="2002-2014 The Neverball Team"
+COPYRIGHT="2002-2019 The Neverball Team"
 LICENSE="GNU GPL v2"
-REVISION="5"
-SOURCE_URI="https://neverball.org/neverball-$portVersion.tar.gz"
-CHECKSUM_SHA256="73fe63cca4f96e2d355480d03bc0b2904e83a0abdf65fe8c52db5cc3cca88fa0"
+REVISION="1"
+srcGitRev="cd7dfc9464e242f100dabe711863f3a943cec1d5"
+SOURCE_URI="https://github.com/Neverball/neverball/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="ee1b9ffb4c32d0e1c0993f5900c94bbc35d5b9e836e487c7a4aa71c547c1e87a"
+SOURCE_FILENAME="neverball-$srcGitRev-$portVersion.tar.gz"
```

Comment:
This probably needs to be reverted, first $portVersion then $srcGitRev?

Reply:
yes